modules/stb_vorbis/audio_stream_ogg_vorbis.cpp

```diff
@@ -31,7 +31,9 @@
 
 #include "os/file_access.h"
 
+#pragma GCC diagnostic ignored "-Wmaybe-uninitialized"
```

Comment:
Might be worth making a PR upstream if there are initialized variables in stb_vorbis: https://github.com/nothings/stb

Reply:
I'll do that, once that's done I'll make another PR here to remove this pragma again.